packages/ckeditor5-list/tests/documentlistproperties/documentlistpropertiesediting.js

```diff
@@ -932,5 +932,68 @@ describe( 'DocumentListPropertiesEditing', () => {
 				` ) );
 			} );
 		} );
+
+		describe( 'upcasting', () => {
```

Comment:
Please move conversion tests to `packages/ckeditor5-list/tests/documentlistproperties/converters.js`.